modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1252,6 +1252,106 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user can use Formula Field with Email Notification"
+	@priority = 4
+	test CanFormulaFieldBeUsedWithEmailNotification {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: A notification template related to Custom Object that contains a Notification action") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Email Template",
+				notificationType = "Email",
+				subject = "Subject Test");
+
+			PortletEntry.save();
+
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176849",
+				objectName = "CustomObject176849",
+				pluralLabelName = "Custom Objects 176849");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176849");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Integer 1",
+				fieldType = "Integer");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field",
+				fieldType = "Formula",
+				formulaFieldOption = "Integer");
+
+			ObjectAdmin.goToFieldsDetails(label = "Custom Formula Field");
+
+			Type(
+				key_labelName = "Formula Builder",
+				locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT",
+				value1 = "customInteger1 + customInteger1");
+
+			Button.clickSave();
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject176849");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176849");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Notification action when entry is added",
+				actionName = "notificationActionWhenEntryIsAdded",
+				active = "true",
+				inputExpression = "customFormulaField == 30",
+				notificationName = "Email Template",
+				thenAction = "Notification",
+				whenAction = "On After Add");
+		}
+
+		task ("When: An entry is added") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customInteger1",
+				objectName = "CustomObject176849",
+				value = 15);
+		}
+
+		task ("Then: The action is triggered and the notification is sent") {
```

Comment:
Since in this test an email is sent, this case requires the use of `MockMock.macro`, adding the property `test.smtp.server.enabled = "true";` to the test.

Reply:
Just to keep this updated, as we discussed in the team , this won't be necessary for this case. Thanks anyway @davisaints !